src/actions/auth_actions/confirm_login_action.js

```diff
@@ -12,7 +12,7 @@ Actions.register(CONFIRM_LOG_IN, payload => {
   })
   .then(checkStatus)
   .then(response => {
-    authStore.setAuthed(response.data.authToken);
+    authStore.setAuthed(response.data.token);
```

Comment:
Does this relate to changing how we do auth here? Trying to follow this piece, but not sure why the change from authToken